update/ansible/playbook/tasks/files/datasources.yml

```diff
@@ -48,3 +40,9 @@ datasources:
   url: http://localhost:9093/alertmanager/
   jsonData:
     keepCookies: []
+- name: ClickHouse
```

Comment:
I believe we need to delete the previous one by inserting these lines at the very top:
```
deleteDatasources:
  - name: ClickHouse
    orgId: 1
```